app/mailers/application_mailer.rb

```diff
@@ -1,5 +1,5 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'jobs@devcongress.org',
+  default from: 'DevCongress Jobs',
```

Comment:
@claudey Shouldn't this be an email address too? If we want to include the name we could change it to `DevCongress Jobs <jobs@devcongress.org>` instead.

Reply:
@claudey I don't mind if you push directly to `master` to restore order to the universe.